common/rakelib/changelog.rake

```diff
@@ -3,15 +3,15 @@
 
 namespace :changelog do
   desc 'Add the empty CHANGELOG entries after a new release'
-  task :reset do |task|
+  task :reset do |_task|
```

Comment:
If you don't use any of the parameters you can also completely remove them  (`do` instead of `do |task|`)